addons/mail/static/src/widgets/form_renderer/form_renderer.scss

```diff
@@ -11,6 +11,19 @@
     width: 100%;
 }
 
+.o_form_sheet_bg .o_FormRenderer_chatterContainer {
+    max-width: none;
+    background-color: #FFF;
+    width: auto;
+    margin-left: -16px;
+    margin-right: -16px;
+
+    .o_ChatterContainer {
+        max-width: $o-form-view-sheet-max-width;
+        margin: 0 auto;
+    }
+}
```

Comment:
Same question as always, are those overrides, and if so, from which original values?
Can't we make it so that the "wrong values" are not set in this case?

Reply:
In this case, the chatter is placed inside the .o_form_sheet_bg witch force a padding left and right (16px).
In 13.3 the chatter was bugged if the screen was larger than 1920px.

So I use the `.o_FormRenderer_chatterContainer` to build a white background that stretch to edge and keep the `.o_ChatterContainer` to the max-with.

13.3: 
![image](https://user-images.githubusercontent.com/1810149/93742704-df939200-fbee-11ea-98f0-15995c6dff74.png)

13.4 (with this fix): 
![image](https://user-images.githubusercontent.com/1810149/93742739-f2a66200-fbee-11ea-9b5b-79efe174af4a.png)

As for the override: 
`max-width: none;` override the `.o_FormRenderer_chatterContainer` max-with so it can be expended to the right and left.
The background is new and other chatter (those who are not inside the `.o_form_sheed_bg`) don't need it.
`width: auto;` will expend the `.o_FormRenderer_chatterContainer` to the edge. We can't use the initial 100% value because it will use the `.o_form_sheet_bg` width value.
margin-left and right -16px will neutralize the .o_form_sheet_bg 16px paddings.

Then the  `.o_ChatterContainer` take the role of the `.o_FormRenderer_chatterContainer` by sizing it (max-width) and center the content (margin).

Reply:
As discussed, you need to make it so that the problematic rule are not loaded in the first place, and avoid override, especially with a specificity selector such as `.o_form_sheet_bg .o_FormRenderer_chatterContainer`.

This will probably imply to add a modifier class in `o_FormRenderer_chatterContainer` at the appropriate time (community and enterprise to check), have a `not` on that class for the rules that are problematic here.

I think the only case that is problematic here is "the inverse of o-aside (in enterprise) but only when the reason is because there is attachmentPreview".

I don't think we even want to change the other cases of "inside o_form_sheet_bg".